modules/installation-configuration-parameters.adoc

```diff
@@ -1,54 +1,110 @@
 // Module included in the following assemblies:
 //
-// * installing-AWS/installing-customizations-cloud.adoc
+// * installing-aws/installing-customizations-cloud.adoc
 
 [id='installation-configuration-parameters_{context}']
 = Installation configuration parameters
 
-You use a `*./install-config.yaml*` file to provide parameter values during a
-{product-title} installation. During the initialization phase of installation,
-you provide values for parameters that are unique to your cluster through the
-command line. You can also modify the default values of the remaining parameters.
+Before you deploy a {product-title} cluster, you provide parameter values to 
+describe your Amazon Web Services (AWS) account and optionally customize your
+cluster's platform.  During the initialization phase of installation, you
+provide values for the required parameters through the command line. If you
+customize your cluster, you can modify the `./install-config.yaml` file to
+provide more details about the platform.
 
 [NOTE]
 ====
 You cannot modify these parameters after installation.
 ====
 
-.Parameters that you define during initialization
-[cols=".^2,.^3,.^3a",options="header"]
+.Required parameters
+[cols=".^2,.^3,.^5a",options="header"]
 |====
 |Parameter|Description|Values
 
 |`provider`
 |The cloud provider to host the cluster.
-a|Enter one of the following values:
+|`aws`
+
+|`baseDomain`
+|The base domain of your cloud provider. This value is used to create routes
+to your {product-title} cluster components.
+|A fully-qualified domain or subdomain name, such as `dev.example.com`.
```

Comment:
That is a tongue twister. So, when we tell users to set up their domain or subdomain, is there extra guidance that we should give them? Like, you might not want to create separate subdomains for your different clusters because the route is a combination of the cluster name and domain?